### DISCUSSION TITLE
Current Status of Project

### DISCUSSION THREAD

**Discussion by 72gm** [TanStack/table#3627] [General]
HI,

I'm looking to evaluate this product and I noticed the following: 

- There are a number of unactioned open issues and discussion items. 
- There hasn't been an update to the source for 9 months

I was wondering: Is this product still being actively maintained? Are there likely to be further releases?

Regards, Graeme

**Comment by Moshyfawn** [comment]
There's an ongoing effort to release the next major React-Table version V8 alpha soon
https://twitter.com/tan_stack/status/1481000372256731145
